code/part-one/blockchain.js

```diff
@@ -31,16 +37,15 @@ class Block {
 
     **Hint**
     Unreplicable data can include
-      - the block index
+      - transactions
       - the previous hash
       - the block timestamp
   */
   calculateHash() {
     return SHA512(`
-      ${this.index}
       ${this.previousHash}
       ${this.timestamp}
-      ${JSON.stringify(this.data)}
+      ${JSON.stringify(this.transactions)}
       ${this._nonce}
     `).toString();
```

Comment:
Left this common on an outdated diff, so I'll add it again here:

This use of a template string seems problematic to me. Since the newlines and indents will be included in the string, if someone were to refactor the use of whitespace later, it would break the blockchain. I would prefer concatenation here.